src/Core/MailTemplate/FolderThemeScanner.php

```diff
@@ -190,6 +190,16 @@ private function checkThemeFolder($mailThemeFolder)
 
     private function getTemplatePath(SplFileInfo $fileInfo): string
     {
-        return '@MailThemes' . substr($fileInfo->getRealPath(), strlen($this->baseThemeFolder));
+        $realPath = $fileInfo->getRealPath();
+
+        if (_PS_VERSION_ >= '8.0.0') {
```

Comment:
I'm not sure to understand the purpose of this check 🤔

Reply:
Before PS8 this change is not needed and would even probably break the rendering of the templates (I did not tested tbh) so I'm enforcing it only for PS8+

Reply:
I think it's not needed as this code won't be introduced before PrestaShop 8.1

Reply:
Done